tola/util.py

```diff
@@ -188,27 +190,35 @@ def get_column_types(rows):
 
     return col_type, data
 
-def cleanDataObj(obj, silo):
+
+def clean_data_obj(obj, col_type={}):
```

Comment:
Since we are basically handling raw user input here either `clean_data_object` or `cleanKey` should validate the input to prevent mongodb or other possible injections. Or are we handling this issue in another function?